tests/io_test.py

```diff
@@ -195,6 +214,7 @@ def fetchMock(url, request):
                 task, inputs=copy.deepcopy(inputs), cleanup=False,
                 validate=False, auto_convert=False)
             self.assertEqual(out['y']['data'], 'dummy file contents_suffix')
+            """
```

Comment:
Shall we remove this testing that is now commented out or were you going to fix it up and keep it?